src/google/protobuf/compiler/parser_unittest.cc

```diff
@@ -2808,6 +2808,34 @@ TEST_F(SourceInfoTest, Fields) {
   EXPECT_TRUE(HasSpan(file_.message_type(0), "name"));
 }
 
+TEST_F(SourceInfoTest, Fields) {
```

Comment:
I think the test is failing because this test case has the same name as the previous case.

Reply:
Yep. I was having problems getting `make check` to run locally. I just got it running locally and verified that the test passes. (I also intentionally commented out a line below, hoping it would make the test fail, to ensure that these assertions would catch any location for the label that should _not_ be present.)

Anyhow, it should be good now.

Reply:
BTW, where did you see test output with that error? The only things I see in GitHub are the CLA check and a "mergeable" check which was looking at PR labels that I can't set. Or did a third check appear on a prior commit which will eventually show up for the last commit I pushed?

Reply:
I think the test results must have disappeared after you pushed a new commit (since they would be invalidated in that case). I just added the tag to kick off a new test run though, so the results should start reappearing.